Changes

```diff
@@ -2,6 +2,12 @@ Revision history for Perl module Mojo::WebSocketProxy
 
 {{$NEXT}}
 
+0.12      2019-05-06 16:57:54+08:00 Asia/Minila
```

Comment:
With this line and version generated, has the resulting dist been uploaded to PAUSE already?

Reply:
Also I don't see any corresponding `Makefile.PL` changes

Reply:
our bad, first time working on public module, didnt know that we should upgrade the versioning in another PR with `dzil release`. We somehow did this manually, removing it right now 😅